### PR TITLE
Dev ts

### PR DESCRIPTION
**Pull request by kwolosonovich**
Coding challenge submission

### CODE REVIEW

**Comment by kwolosonovich** [comment]
@tomboolean Coding challenge

File: src/config/connection/connection.ts [hunk 1]
```diff
@@ -2,18 +2,21 @@ import * as mongoose from 'mongoose';
 import config from '../env/index';
 
 interface IConnectOptions {
-    autoReconnect: boolean;
-    reconnectTries: number; // Never stop trying to reconnect
-    reconnectInterval: number;
+    // deprecated 5.7.14 - auto reconnect defaults true
```
Comment (tomboolean) [inline src/config/connection/connection.ts:5]: nit - commented code

File: src/config/connection/connection.ts [hunk 2]
```diff
@@ -2,18 +2,21 @@ import * as mongoose from 'mongoose';
...
+    useUnifiedTopology ? : boolean;
 }
 
 const connectOptions: IConnectOptions = {
-    autoReconnect: true,
-    reconnectTries: Number.MAX_VALUE,
-    reconnectInterval: 1000,
+    // autoReconnect: true,
+    // reconnectTries: Number.MAX_VALUE,
+    // reconnectInterval: 1000,
```
Comment (tomboolean) [inline src/config/connection/connection.ts:17]: ditto

File: src/routes/StackRouter.ts [hunk 3]
```diff
@@ -0,0 +1,185 @@
...
+ *    responses:
+ *      200:
+ *        description: return updated stack
+ *        content:
+ *          application/json:
+ *            schema:
+ *              oneOf:
+ *                - $ref: '#/components/schemas/StackSchema'
+ */
+router.patch('/:id', StackComponent.update);
```
Comment (tomboolean) [inline src/routes/StackRouter.ts:144]: SWEET - you did PATCH too!

**Review by tomboolean — APPROVED** [review]
GREAT job! Very impressed

**Pull request closed without merging**